### ISSUE TITLE
Magerun 7.0.0+ prints out short & long form parameters as the long form | Does not recognise shortform alias

### ISSUE THREAD

**Issue by furan917** [netz98/n98-magerun2#1295]
## Describe the bug

If you run a command that has the wrong argument in it, or in at least potentially one case the right argument, n98-magerun returns a list of allowed arguments however both the - & -- have the same text.

## Expected behaviour

If I run `n98-magerun setup:static-content:deploy -fakeParamHere` it should return the expected option doesn't exist error with an output of the command and allowed/expected arguments.
```
setup:static-content:deploy [-f|--force] [-s|--strategy [STRATEGY]] [-a|--area [AREA]] [--exclude-area [EXCLUDE-AREA]] [-t|--theme [THEME]] [--exclude-theme [EXCLUDE-THEME]] [-l|--language [LANGUAGE]] [--exclude-language [EXCLUDE-LANGUAGE]] [-j|--jobs [JOBS]] [--max-execution-time [MAX-EXECUTION-TIME]] [--symlink-locale] [--content-version CONTENT-VERSION] [--refresh-content-version-only] [--no-javascript] [--no-js-bundle] [--no-css] [--no-less] [--no-images] [--no-fonts] [--no-html] [--no-misc] [--no-html-minify] [--no-parent] [--] [<languages>...]
```

However when I run that, at least on all v7 version, I get the following output
```
setup:static-content:deploy [-force|--force] [-strategy|--strategy [STRATEGY]] [-area|--area [AREA]] [-exclude-area|--exclude-area [EXCLUDE-AREA]] [-theme|--theme [THEME]] [-exclude-theme|--exclude-theme [EXCLUDE-THEME]] [-language|--language [LANGUAGE]] [-exclude-language|--exclude-language [EXCLUDE-LANGUAGE]] [-jobs|--jobs [JOBS]] [-max-execution-time|--max-execution-time [MAX-EXECUTION-TIME]] [-symlink-locale|--symlink-locale] [-content-version|--content-version [CONTENT-VERSION]] [-refresh-content-version-only|--refresh-content-version-only] [-no-javascript|--no-javascript] [-no-js-bundle|--no-js-bundle] [-no-css|--no-css] [-no-less|--no-less] [-no-images|--no-images] [-no-fonts|--no-fonts] [-no-html|--no-html] [-no-misc|--no-misc] [-no-html-minify|--no-html-minify] [-no-parent|--no-parent] [--] [<languages>...]
```

as you can see, it doesn't correctly print the right shorthand.

Note: Expected output was from a run via bin/magento and the second one via n98-magerun on the same system and install running M2 v2.4.5-p2

## Steps to reproduce the issue
Install any version from v7.0.0 to v7.1.0 and run `n98-magerun setup:static-content:deploy -fakeParamHere` to purposely generate the error message. 

## Technical details
Replicated on Ubuntu 22/20, AmazonLinux 1/2 and Mac M2 venture 13.5.2. On all v7 tags, and testest on 2.4.4, 2.4.5-p2 and 2.4.6 

## Additional context

Also, I am still digging into it but also try:
`./n98-magerun2.phar setup:static-content:deploy -f` (any v7 variant)
vs
`./n98-magerun2-6.1.1.phar setup:static-content:deploy -f`
vs
`bin/magento setup:static-content:deploy -f`

Let me know if you also get magerun saying `"-f"` option does not exist but the bin/magento works or the `"-s"`

**Comment by cmuench** [comment]
@furan917 The issues should be fixed. You can update your phar file to develop version by running `n98-magerun2.phar self-update --unstable` to verify if it works with the short options.

Your example with `setup:static-content:deploy -s` should work now. It's registering the shortcut option (now).

The `-fakeParam` is not validated in the core command.

**Comment by furan917** [comment]
@cmuench Amazing! Thank you. 

I can confirm that the tested `-s` and `-f` arguments for `setup:static-content:deploy` now work correctly, and that when printing arguments it now prints out the correct short/long form.

This was the purpose of the fakeparam in testing, to validate the we get the correct error response I used `setup:static-content:deploy -f --ssh`

This now prints out 
```

  The "--ssh" option does not exist.

setup:static-content:deploy [-f|--force] [-s|--strategy [STRATEGY]] [-a|--area [AREA]] [--exclude-area [EXCLUDE-AREA]] [-t|--theme [THEME]] [--exclude-theme [EXCLUDE-THEME]] [-l|--language [LANGUAGE]] [--exclude-language [EXCLUDE-LANGUAGE]] [-j|--jobs [JOBS]] [--max-execution-time [MAX-EXECUTION-TIME]] [--symlink-locale] [--content-version [CONTENT-VERSION]] [--refresh-content-version-only] [--no-javascript] [--no-js-bundle] [--no-css] [--no-less] [--no-images] [--no-fonts] [--no-html] [--no-misc] [--no-html-minify] [--no-parent] [--] [<languages>...]
```

We ignore the --shh option does not exist because, well, we purposely are generating this error haha. But we can see the alias's are not correct.

**Comment by cmuench** [comment]
@furan917 Great. I will deploy that fix then with the upcoming 7.2.0 release.

**Comment by cmuench** [comment]
Will be released, soon.
